cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/IOUContract.kt

```diff
@@ -35,23 +35,23 @@ class IOUContract : Contract {
         when (command.value) {
             is Commands.Create -> requireThat {
                 // Generic constraints around the IOU transaction.
-                "No inputs should be consumed when issuing an IOU." using (tx.inputs.isEmpty())
-                //Only one output state should be created
+                "No IOUState inputs should be consumed when issuing an IOU." using (tx.inputsOfType<IOUState>().isEmpty())
+                "There should be only one IOUState output." using (tx.outputsOfType<IOUState>().size == 1)
                 val out = tx.outputsOfType<IOUState>().single()
                 "The lender and the borrower cannot be the same entity." using (out.lender != out.borrower)
+                "Command require two signers." using (command.signers.size == 2)
                 "All of the participants must be signers." using (command.signers.containsAll(out.participants.map { it.owningKey }))
```

Comment:
By using `participants` here you are dependent on its implementation. To be more strict, you could use `listOf(lender, borrower)`. True, it is the same thing **at this moment**.